docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-planning-problem-con.adoc

```diff
@@ -0,0 +1,53 @@
+[id='optimizer-planning-problem-con']
+= What is a planning problem?
+
+
+A planning problem has an optimal goal, based on limited resources and under specific constraints. Optimal goals can be any number of things, such as:
+
+* Maximized profits - the optimal goal results in the highest possible profit.
+* Minimized ecological footprint - the optimal goal has the least amount of environmental impact.
+* Maximized satisfaction for employees or customers - the optimal goal prioritizes the needs of employees or customers.
+
+The ability to achieve these goals relies on the number of resources available, such as:
+
+* The number of people
+* Amount of time
+* Budget
+* Physical assets, for example, machinery, vehicles, computers, buildings, etc
+
+Specific constraints related to these resources must also be taken into account, such as the number of hours a person works, their ability to use certain machines, or compatibility between pieces of equipment. 
+ 
+The Business Optimizer helps Java programmers solve constraint satisfaction problems efficiently. Under the hood, it combines optimization heuristics and metaheuristics with efficient score calculation.
+
+
+[id='optimizer-planning-problem-search-space-con']
+== A Planning Problem Has a Huge Search Space
+
+A planning problem has a number of solutions.
+
+There are several categories of solutions:
+
+Possible solution::
+A possible solution is any solution, whether or not it breaks any number of constraints. Planning problems tend to have an incredibly large number of possible solutions. Many of those solutions are worthless.
+
+Feasible solution::
+A feasible solution is a solution that does not break any (negative) hard constraints. The number of feasible solutions tends to be relative to the number of possible solutions. Sometimes there are no feasible solutions. Every feasible solution is a possible solution.
+
+Optimal solution::
+An optimal solution is a solution with the highest score. Planning problems tend to have 1 or a few optimal solutions. There is always at least 1 optimal solution, even in the case that there are no feasible solutions and the optimal solution is not feasible.
```

Comment:
tend to - casual. Consider 'usually' or 'often'. For numbers under 10, spell out (one). Consider' Planning problems usually have a few options solutions. They always have at least one..

Reply:
oh man, I thought I got rid of all of the numbers, but clearly a few slipped through!